editor/src/components/canvas/__snapshots__/ui-jsx-canvas-bugs.spec.tsx.snap

```diff
@@ -0,0 +1,33 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`UiJsxCanvas #747 - DOM object constructor cannot be called as a function 1`] = `
```

Comment:
I think this test is small enough to be inlined!

Reply:
Can't easily be done given that `testCanvasRender` checks a bunch of different things.

Reply:
there's `testCanvasRenderInline` to solve that problem